webflow-hacks/misc-hacks/13/13.ts

```diff
@@ -0,0 +1,26 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const BACK_ONE_SELECTOR = '[fs-hacks-element="hack13-back-1"]';
+  const BACK_TWO_SELECTOR = '[fs-hacks-element="hack13-back-2"]';
+
+  // on click
+  const backOne = document.querySelector<HTMLDivElement>(BACK_ONE_SELECTOR);
+  const backTwo = document.querySelector<HTMLDivElement>(BACK_TWO_SELECTOR);
+  if (!backOne || !backTwo) return;
```

Comment:
Now this is assuming that users need to have both backOne and backTwo buttons on the page.
But users will probably will just implement either one or the other, but not both, right?

Can we split this in to different functions, so users can choose from?

Reply:
So two different hacks @alexiglesias93?

Reply:
Let's keep it like this, two hacks will confuse users more